include/dice/template-library/polymorphic_allocator.hpp

```diff
@@ -118,9 +118,9 @@ namespace dice::template_library {
 		using void_pointer = typename detail_pmr::same_type<typename std::allocator_traits<Allocators<T>>::void_pointer...>::type;
 		using const_void_pointer = typename detail_pmr::same_type<typename std::allocator_traits<Allocators<T>>::const_void_pointer...>::type;
 
-		using propagate_on_container_copy_assignment = std::bool_constant<(std::allocator_traits<Allocators<T>>::propagate_on_container_copy_assignment::value || ...)>;
-		using propagate_on_container_move_assignment = std::bool_constant<(std::allocator_traits<Allocators<T>>::propagate_on_container_move_assignment::value || ...)>;
-		using propagate_on_container_swap = std::bool_constant<(std::allocator_traits<Allocators<T>>::propagate_on_container_swap::value || ...)>;
+		using propagate_on_container_copy_assignment = std::bool_constant<(std::allocator_traits<Allocators<T>>::propagate_on_container_copy_assignment::value && ...)>;
```

Comment:
for these constants I changed `or` to `and`, because of things I read about `std::polymorphic_allocator` but I'm still not entirely sure which of these is the correct one

Reply:
As discussed, `and` is the conservative approach that should always work. We should still ask Keita if metall can set the propagation constants to true.

Reply:
As discussed: `&&` is the more conservative option, so this is what we are going to use